src/ralph/admin/static/elements/auto-complete.html

```diff
@@ -151,9 +158,12 @@
           value: function(){return [];}
         }
       },
-      handleTyping: function(evt, obj) {
-        console.log('handleTyping', evt, obj);
-        if (evt.target.value === '' || evt.target.value.length < this.fireThreshold) return false;
+      sendQueryRequest: function(evt, obj) {
+        console.log('sendQueryRequest', evt, obj);
```

Comment:
i thought about removing it in some gulp action. But do we have versioned static html files/templates? :)